<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowMeb</class>
 <widget class="QMainWindow" name="MainWindowMeb">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindowMeb</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="labBorder2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>59</y>
      <width>500</width>
      <height>291</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">position: absolute;
width: 613px;
height: 104px;
left: 34px;
top: 112px;

background: #C4C4C4;
border-radius: 20px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lbIDNum">
    <property name="geometry">
     <rect>
      <x>269</x>
      <y>476</y>
      <width>381</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">/*Border 2 */


position: absolute;
width: 560px;
height: 45px;
left: 53px;
top: 142px;

background: #EDEDED;
border-radius: 5px;

/*Text*/

position: absolute;
width: 305px;
height: 31px;
left: 115px;
top: 153px;

font-family: Roboto;
font-style: normal;
font-weight: normal;
font-size: 14px;
line-height: 22px;
/* or 92% */

letter-spacing: -0.408px;

color: #605E5E;</string>
    </property>
    <property name="text">
     <string>ID Number</string>
    </property>
    <property name="margin">
     <number>3</number>
    </property>
   </widget>
   <widget class="QLabel" name="lbTitle">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>372</y>
      <width>430</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">/*Border 2 */


position: absolute;
width: 560px;
height: 45px;
left: 53px;
top: 142px;

background: #EDEDED;
border-radius: 5px;

/*Text*/

position: absolute;
width: 305px;
height: 31px;
left: 115px;
top: 153px;

font-family: Roboto;
font-style: normal;
font-weight: normal;
font-size: 14px;
line-height: 22px;
/* or 92% */

letter-spacing: -0.408px;

color: #605E5E;</string>
    </property>
    <property name="text">
     <string>Title of Book</string>
    </property>
    <property name="margin">
     <number>3</number>
    </property>
   </widget>
   <widget class="QLabel" name="lbDate">
    <property name="geometry">
     <rect>
      <x>279</x>
      <y>441</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">/*Border 2 */


position: absolute;
width: 560px;
height: 45px;
left: 53px;
top: 142px;

background: #EDEDED;
border-radius: 5px;

/*Text*/

position: absolute;
width: 305px;
height: 31px;
left: 115px;
top: 153px;

font-family: Roboto;
font-style: normal;
font-weight: normal;
font-size: 14px;
line-height: 22px;
/* or 92% */

letter-spacing: -0.408px;

color: #605E5E;</string>
    </property>
    <property name="text">
     <string>Date of Book</string>
    </property>
    <property name="margin">
     <number>3</number>
    </property>
   </widget>
   <widget class="QLabel" name="lbAvailability">
    <property name="geometry">
     <rect>
      <x>269</x>
      <y>511</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">/*Border 2 */


position: absolute;
width: 560px;
height: 45px;
left: 53px;
top: 142px;

background: #EDEDED;
border-radius: 5px;

/*Text*/

position: absolute;
width: 305px;
height: 31px;
left: 115px;
top: 153px;

font-family: Roboto;
font-style: normal;
font-weight: normal;
font-size: 14px;
line-height: 22px;
/* or 92% */

letter-spacing: -0.408px;

color: #605E5E;</string>
    </property>
    <property name="text">
     <string>Book Availability</string>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
    <property name="margin">
     <number>3</number>
    </property>
    <property name="indent">
     <number>-1</number>
    </property>
   </widget>
   <widget class="QLabel" name="labelBookTitleText">
    <property name="geometry">
     <rect>
      <x>158</x>
      <y>371</y>
      <width>51</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">/* Name: */


position: absolute;
width: 104px;
height: 28px;
left: 65px;
top: 136px;

font-family: Roboto;
font-style: normal;
font-weight: normal;
font-size: 18px;
line-height: 22px;
/* or 92% */

text-align: center;
letter-spacing: -0.408px;</string>
    </property>
    <property name="text">
     <string>Title:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lbGenreTxt">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>440</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">/*Border 2 */


position: absolute;
width: 560px;
height: 45px;
left: 53px;
top: 142px;

background: #EDEDED;
border-radius: 5px;

/*Text*/

position: absolute;
width: 305px;
height: 31px;
left: 115px;
top: 153px;

font-family: Roboto;
font-style: normal;
font-weight: normal;
font-size: 14px;
line-height: 22px;
/* or 92% */

letter-spacing: -0.408px;

color: #605E5E;
</string>
    </property>
    <property name="text">
     <string>Genre</string>
    </property>
   </widget>
   <widget class="QLabel" name="labBookCover">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>370</y>
      <width>121</width>
      <height>171</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 10px;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelPublishText">
    <property name="geometry">
     <rect>
      <x>159</x>
      <y>441</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">/* Name: */


position: absolute;
width: 104px;
height: 28px;
left: 65px;
top: 136px;

font-family: Roboto;
font-style: normal;
font-weight: normal;
font-size: 18px;
line-height: 22px;
/* or 92% */

text-align: center;
letter-spacing: -0.408px;</string>
    </property>
    <property name="text">
     <string>Publish Date:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelIDnoText">
    <property name="geometry">
     <rect>
      <x>159</x>
      <y>476</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">/* Name: */


position: absolute;
width: 104px;
height: 28px;
left: 65px;
top: 136px;

font-family: Roboto;
font-style: normal;
font-weight: normal;
font-size: 18px;
line-height: 22px;
/* or 92% */

text-align: center;
letter-spacing: -0.408px;</string>
    </property>
    <property name="text">
     <string>ID Number:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lbAuthor">
    <property name="geometry">
     <rect>
      <x>239</x>
      <y>406</y>
      <width>411</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">/*Border 2 */


position: absolute;
width: 560px;
height: 45px;
left: 53px;
top: 142px;

background: #EDEDED;
border-radius: 5px;

/*Text*/

position: absolute;
width: 305px;
height: 31px;
left: 115px;
top: 153px;

font-family: Roboto;
font-style: normal;
font-weight: normal;
font-size: 14px;
line-height: 22px;
/* or 92% */

letter-spacing: -0.408px;

color: #605E5E;</string>
    </property>
    <property name="text">
     <string>Author of Book</string>
    </property>
    <property name="margin">
     <number>3</number>
    </property>
   </widget>
   <widget class="QLabel" name="labelAuthorText">
    <property name="geometry">
     <rect>
      <x>157</x>
      <y>406</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">/* Name: */


position: absolute;
width: 104px;
height: 28px;
left: 65px;
top: 136px;

font-family: Roboto;
font-style: normal;
font-weight: normal;
font-size: 18px;
line-height: 22px;
/* or 92% */

text-align: center;
letter-spacing: -0.408px;</string>
    </property>
    <property name="text">
     <string>Author:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelAvailabilityText">
    <property name="geometry">
     <rect>
      <x>164</x>
      <y>511</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">/* Name: */


position: absolute;
width: 104px;
height: 28px;
left: 65px;
top: 136px;

font-family: Roboto;
font-style: normal;
font-weight: normal;
font-size: 18px;
line-height: 22px;
/* or 92% */

text-align: center;
letter-spacing: -0.408px;</string>
    </property>
    <property name="text">
     <string>Availability:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labBorder3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>360</y>
      <width>661</width>
      <height>191</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">position: absolute;
width: 613px;
height: 104px;
left: 34px;
top: 112px;

background: #C4C4C4;
border-radius: 20px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pbRefresh">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>13</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">/*Border 2 */


position: absolute;
width: 560px;
height: 45px;
left: 53px;
top: 142px;

background: #C4C4C4;
border-radius: 5px;

/*Text*/

position: absolute;
width: 305px;
height: 31px;
left: 115px;
top: 153px;

font-family: Roboto;
font-style: normal;
font-weight: normal;
font-size: 14px;
line-height: 22px;
/* or 92% */

letter-spacing: -0.408px;

color: #605E5E;</string>
    </property>
    <property name="text">
     <string>Refresh</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lEditSearchBook1">
    <property name="geometry">
     <rect>
      <x>18</x>
      <y>20</y>
      <width>480</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">/*Border 2 */


position: absolute;
width: 560px;
height: 45px;
left: 53px;
top: 142px;

background: #EDEDED;
border-radius: 10px;

/*Text*/

position: absolute;
width: 305px;
height: 31px;
left: 115px;
top: 153px;

font-family: Roboto;
font-style: normal;
font-weight: normal;
font-size: 18px;
line-height: 22px;
/* or 92% */

letter-spacing: -0.408px;

color: #868181;
</string>
    </property>
    <property name="text">
     <string>Search Books</string>
    </property>
   </widget>
   <widget class="QLabel" name="labBorder1">
    <property name="geometry">
     <rect>
      <x>8</x>
      <y>10</y>
      <width>500</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">/*Border 3*/

position: absolute;
width: 613px;
height: 104px;
left: 34px;
top: 112px;

background: #C4C4C4;
border-radius: 20px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelGenre">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>440</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">/* Name: */


position: absolute;
width: 104px;
height: 28px;
left: 65px;
top: 136px;

font-family: Roboto;
font-style: normal;
font-weight: normal;
font-size: 18px;
line-height: 22px;
/* or 92% */

text-align: center;
letter-spacing: -0.408px;</string>
    </property>
    <property name="text">
     <string>Genre:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QListWidget" name="list">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>481</width>
      <height>271</height>
     </rect>
    </property>
   </widget>
   <zorder>labBorder3</zorder>
   <zorder>labBorder1</zorder>
   <zorder>labBorder2</zorder>
   <zorder>lbIDNum</zorder>
   <zorder>lbTitle</zorder>
   <zorder>lbDate</zorder>
   <zorder>lbAvailability</zorder>
   <zorder>labelBookTitleText</zorder>
   <zorder>lbGenreTxt</zorder>
   <zorder>labBookCover</zorder>
   <zorder>labelPublishText</zorder>
   <zorder>labelIDnoText</zorder>
   <zorder>lbAuthor</zorder>
   <zorder>labelAuthorText</zorder>
   <zorder>labelAvailabilityText</zorder>
   <zorder>pbRefresh</zorder>
   <zorder>lEditSearchBook1</zorder>
   <zorder>labelGenre</zorder>
   <zorder>list</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuProfile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionProfile"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuProfile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionProfile">
   <property name="text">
    <string>Profile</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
